index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Tesla</title>
+	<meta charset="UTF-8">
+	<link rel="stylesheet" href="/style.css">
+</head>
+	<body>
+		<header>
+			<div class="menu">
+				<nav class="menu__bar">
+					<img src="" alt="tesla_wordmark_red" class=menu__bar_watermark>
+					<a href="" class="menu__bar_item">Model S</a>
```

Comment:
> `menu__bar_item`
I'm not sure that `_item` is a modifier here. I'd say it makes more sense to do `menu__bar-item` here